lib/lfrfid/protocols/protocol_fdx_b.c

```diff
@@ -348,6 +360,10 @@ void protocol_fdx_b_render_brief_data(ProtocolFDXB* protocol, FuriString* result
     } else {
         furi_string_cat(result, "---");
     }
+
+    free((char*)country.two_letter);
```

Comment:
Sketchy af.
If your returning container then provide methods to handle it. I'd make ISO3166Country anonymous structure and provide methods to get each field.